tests/test_referral.py

```diff
@@ -9,6 +9,7 @@
 REFERRAL_USER_PROD_API_KEY = os.getenv("REFERRAL_USER_PROD_API_KEY", "123")
 
 
+# TODO: Rename this file to `test_referral_customer.py` and all test names need `customer` added to them
```

Comment:
Why not do this now?

Reply:
So we can keep the test file matched up to the other file and change them both at the same time.